flink-shaded-jackson-parent/flink-shaded-jackson-2/src/main/resources/META-INF/NOTICE

```diff
@@ -8,7 +8,7 @@ This project bundles the following dependencies under the Apache Software Licens
 
 - com.fasterxml.jackson.core:jackson-annotations:2.13.4
 - com.fasterxml.jackson.core:jackson-core:2.13.4
-- com.fasterxml.jackson.core:jackson-databind:2.13.4
+- com.fasterxml.jackson.core:jackson-databind:2.13.4.2
```

Comment:
Is this really the correct version, looking at what you're referring to in the POM file?

Reply:
Based on issue description in databind repo [1] 
> 2.13.4.2 micro-patch (jackson-bom 2.13.4.20221013). (NOTE: 2.13.4.1/2.13.4.20221012 have an issue that affects Gradle users)

[1] https://github.com/FasterXML/jackson-databind/issues/3590#issue-1362567066

Reply:
also `mvn dependency:tree` for this branch gives
```
[INFO]    +- com.fasterxml.jackson.core:jackson-annotations:jar:2.13.4:compile
[INFO]    +- com.fasterxml.jackson.core:jackson-databind:jar:2.13.4.2:compile
[INFO]    |  \- com.fasterxml.jackson.core:jackson-core:jar:2.13.4:compile
[INFO]    +- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.13.4:compile
[INFO]    |  \- org.yaml:snakeyaml:jar:1.31:compile
[INFO]    +- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:jar:2.13.4:compile

```

Reply:
How freaking annoying...